src/sriov.h

```diff
@@ -19,7 +19,7 @@
 #include "netplan.h"
 
 NETPLAN_INTERNAL void
-write_sriov_conf_finish(const char* rootdir);
+write_sriov_conf_finish(const char* rootdir, GHashTable* sriov_pfs);
```

Comment:
question (blocking, abi): breaking the ABI here used by a binary in `netplan.io`, is this deliberate?

I vaguely recall doing some work in this area, but I don't remember the specifics. More to the point, I don't recall if I tried to maintain ABI with the generate binary or not.

Reply:
Ohh, good spot! I did not do this deliberately, it slipped my mind! We do not want to break the ABI that is used from the `generate` binary (we did that last time, and it was bad).

I've now introduced new API `netplan_state_finish_sriov_write()` and `netplan_sriov_cleanup()` to handle this and moved the old functions to `abi_compat.c`. That should work.

Could you please double check my V2 of commit 9c99162816b6f255173b0adaa906ff9af02b3654 ?